ports/psrl/Phantasy Star RL.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+GAMEDIR="/$directory/ports/psrl"
+
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
+export TEXTINPUTINTERACTIVE="Y"
+
+cd $GAMEDIR
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Rename data.win
+[ -f "./data.win" ] && mv data.win game.droid
+
+$ESUDO chmod 666 /dev/uinput
+
+$GPTOKEYB "gmloadernext" -c "psrl.gptk"&
```

Comment:
```suggestion
$GPTOKEYB "gmloadernext" -c "psrl.gptk" &
```
I'm sure this was working, but I wonder if it'd be a good idea to put a space before the '&' so something goofy doesn't happen on another CFW/device...